sonic-net/sonic-buildimage
# [Nokia-7220 IXR]Update NOKIA 7220 H4-32D

#### Why I did it
Update for NOKIA 7220 H4-32D and NOKIA 7220 H4 to achieve 100% pass in OC for T0/T1 topology
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Added files under ../device/nokia/x86_64-nokia_ixr7220_h4_32d-r0 directory.
Added files under ../device/nokia/x86_64-nokia_ixr7220_h4-r0 directory.
#### How to verify it

1. Make sure the sonic-buildimage is successful
2. Run this image on x86_64-nokia_ixr7220_h4_32d-r0 and x86_64-nokia_ixr7220_h4-r0 and verify all dockers are up and test basic commands like:

- show version
- show platform summary
- show platform syseeprom
- show platform fan
- show platform psustatus
- show platform firmware status
- show platform temperature
- sudo show system-health detail
- show interface status

3. Run OC test, for T0/T1 topology and have 100% pass on platform.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
